### ISSUE TITLE
However you call it (groups, lists, session) it's the same thing, and it's not going to be implemented, as suggested many times before. There are already other extensions that do this.

### ISSUE THREAD

**Issue by Droyk** [hankxdev/one-click-extensions-manager#86]
>However you call it (groups, lists, session) it's the same thing, and it's not going to be implemented, as suggested many times before. There are already other extensions that do this.

_Originally posted by @fregante in https://github.com/hankxdev/one-click-extensions-manager/issues/74#issuecomment-1013573210_

I don't understand this thinking. extension managers also existed prior to your extension manager then why did you create another to do pretty much the same job that they offer? & with ripped features here's an example I'm using this extension atm https://chrome.google.com/webstore/detail/simpleextmanager/kniehgiejgnnpgojkdhhjbgbllnfkfdk?hl=en from way back from 2013. I think It was one of the only few ones that were providing great functionality at the time. (& still, it is.) 
you had everything you wanted then why did you choose to make another one when one already existed with great UI/UX?

What is the point of making an alternative if not making it better than what the current market offers?
& look I'm not forcing you here. I'm just saying your argument for not adding is beyond ridiculous! just because you don't want to add it don't hide behind an argument that doesn't make any sense! If you simply don't want to add it. just say this is my extension manager. I just don't wish to add this feature there's no reason for it but I simply just don't wish to. which is a great reason.
just don't hide behind this shitty argument which doesn't any sense at all!

saying if the other extension manager already does this doesn't mean **there's no reason you shouldn't do it too! make an extension manager that provides it's all!** so users don't need to look any further! other extension managers also had "Click the icon to show installed extensions" prior to your extension manager why couldn't just find a better way to do it? why copy it? why made it all when others already existed? 
- Click extension name to enable/disable this extension
- Right click the extension name to uninstall the extension(with confirm)
- Disable all extensions with one clicking
- Enable all extensions with one clicking
- Open the extension's options page by clicking gear icon after it's name
- Type in the search area to search the extension quickly
- No Ads
- Simple, fast, no background scripts, never read or collect your personal data

all of the above features were there before you even entered the store!

Now I'm not saying copying another extension manager feature is bad if done to make it better there's nothing better than that but in your case, you stagnate the development! if that's what you want that's still fine but say it bluntly don't hide behind an argument that doesn't make any sense!

**Comment by hankxdev** [comment]
wow, such a long issue title and an even longer comment.

@Droyk seems you are curious why we made this extension as there are already many same extensions on the marketplace.

As the original author, let me explain a bit.

Back to 2013 I started making extensions, in order to debug the extension I was creating, I had to disable/enable other extension now and then. So I thought it's better if there was an extension to manager these extensions. There were some indeed, with similar functionalities and UI. However, they either were too complicated  or requires too many permissions, even worse,  some of them inject ads ( although, after several years and Google cleaned the web store many times, now no one injects ads any more)

That's why I created this one my own, simple and fast, and only requires `management` permission. And NO, from UI to code, I did not copy anyone's code, so it's not fair to ask `why copy it`, actually there were some extension copied from this one.

It's only personal interests actually, and then I published it and open sourced it, 

Regarding the question why didn't we add more functionalities to it, because there are many ones has same feature, and even better than this one. this is only a small tool. we just want to keep it small, clean and simple.

Thanks for you suggestion :)
